wwwapp/views.py

```diff
@@ -588,3 +595,31 @@ def page(request):
 
 index_view = as_article("index")
 template_for_workshop_page_view = as_article("template_for_workshop_page")
+
+
+@login_required()
```

Comment:
We might need to adjust it so permission denied is returned on logged out user. I'm more and more temted to test this code on production as I don't have a proper setup with nginx for development.

Reply:
Don't worry, I'll definitely set ot up myself for testing before I give you an approve XD

Reply:
Yeah, you definitely do
```
2019/07/22 10:43:08 [error] 11487#11487: *23 auth request unexpected status: 302 while sending to client, client: ::1, server: , request: "GET /internet/ HTTP/1.1", host: "localhost:8080"
::1 - - [22/Jul/2019:10:43:08 +0200] "GET /internet/ HTTP/1.1" 500 588 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36"
```